<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WTC-5159473-l2" ignored="5" name="MFAC_Test" tests="25" failures="5" timestamp="25 Oct 2018 18:14:13 GMT" time="37.164" errors="0">
  <testcase name="testMethod" time="0.012" classname="TestNG.MFAC"/>
  <testcase name="IssuePin" time="1.631" classname="TestNG.MFAC"/>
  <testcase name="IssuePin" time="2.035" classname="TestNG.MFAC"/>
  <testcase name="IssuePin" time="2.245" classname="TestNG.MFAC"/>
  <testcase name="IssuePin" time="4.778" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinValid" time="6.42" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinValid" time="6.828" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinValid" time="7.485" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinValid" time="7.565" classname="TestNG.MFAC"/>
  <testcase name="IssuePinVelocity" time="9.518" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinNoLongerValid" time="10.595" classname="TestNG.MFAC"/>
  <testcase name="IssuePinVelocity" time="12.001" classname="TestNG.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;5700&amp;quot; and a string containing &amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;)
     but: a string containing &amp;quot;5700&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;65ffa943-381b-4631-baea-175439a11704&amp;quot;,&amp;quot;output&amp;quot;:{&amp;quot;pinOTP&amp;quot;:&amp;quot;055645&amp;quot;,&amp;quot;pinExpirationDate&amp;quot;:&amp;quot;10/25/2018 06:43 PM GMT&amp;quot;}}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "5700" and a string containing "Unfortunately, you have exceeded your attempts for verification. Please try again later.")
     but: a string containing "5700" was "{"transactionId":"65ffa943-381b-4631-baea-175439a11704","output":{"pinOTP":"055645","pinExpirationDate":"10/25/2018 06:43 PM GMT"}}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at TestNG.MFAC.IssuePinVelocity(MFAC.java:224)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- IssuePinVelocity -->
  <testcase name="VerifyPinNoLongerValid" time="13.658" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinNoLongerValid" time="13.714" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinNoLongerValid" time="13.809" classname="TestNG.MFAC"/>
  <testcase name="IssuePinVelocity" time="13.918" classname="TestNG.MFAC"/>
  <testcase name="IssuePinVelocity" time="14.518" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinVelocity" time="18.881" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinVelocity" time="19.029" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinVelocity" time="21.797" classname="TestNG.MFAC"/>
  <testcase name="VerifyPinVelocity" time="22.002" classname="TestNG.MFAC"/>
  <testcase name="AddressVelocity" time="26.354" classname="TestNG.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;DENY&amp;quot; and a string containing &amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;)
     but: a string containing &amp;quot;DENY&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;070c4202-8db3-454d-b076-9185be72fe7c&amp;quot;,&amp;quot;output&amp;quot;:{&amp;quot;advice&amp;quot;:&amp;quot;ALLOW&amp;quot;}}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "DENY" and a string containing "Unfortunately, too many failed attempts for registration have occurred. Please try again later.")
     but: a string containing "DENY" was "{"transactionId":"070c4202-8db3-454d-b076-9185be72fe7c","output":{"advice":"ALLOW"}}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at TestNG.MFAC.AddressVelocity(MFAC.java:186)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="AddressVelocity" time="28.568" classname="TestNG.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;DENY&amp;quot; and a string containing &amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;)
     but: a string containing &amp;quot;DENY&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;7a776ee8-3c85-4194-9ffe-10231455880d&amp;quot;,&amp;quot;output&amp;quot;:{&amp;quot;advice&amp;quot;:&amp;quot;ALLOW&amp;quot;}}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "DENY" and a string containing "Unfortunately, too many failed attempts for registration have occurred. Please try again later.")
     but: a string containing "DENY" was "{"transactionId":"7a776ee8-3c85-4194-9ffe-10231455880d","output":{"advice":"ALLOW"}}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at TestNG.MFAC.AddressVelocity(MFAC.java:186)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="AddressVelocity" time="34.229" classname="TestNG.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;DENY&amp;quot; and a string containing &amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;)
     but: a string containing &amp;quot;DENY&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;4e087e65-77b9-4575-9965-afaaf214e968&amp;quot;,&amp;quot;output&amp;quot;:{&amp;quot;advice&amp;quot;:&amp;quot;ALLOW&amp;quot;}}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "DENY" and a string containing "Unfortunately, too many failed attempts for registration have occurred. Please try again later.")
     but: a string containing "DENY" was "{"transactionId":"4e087e65-77b9-4575-9965-afaaf214e968","output":{"advice":"ALLOW"}}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at TestNG.MFAC.AddressVelocity(MFAC.java:186)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="AddressVelocity" time="35.362" classname="TestNG.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;DENY&amp;quot; and a string containing &amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;)
     but: a string containing &amp;quot;DENY&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;6ddf313c-0071-4ea3-bcc6-4904618b0adb&amp;quot;,&amp;quot;output&amp;quot;:{&amp;quot;advice&amp;quot;:&amp;quot;ALLOW&amp;quot;}}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "DENY" and a string containing "Unfortunately, too many failed attempts for registration have occurred. Please try again later.")
     but: a string containing "DENY" was "{"transactionId":"6ddf313c-0071-4ea3-bcc6-4904618b0adb","output":{"advice":"ALLOW"}}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at TestNG.MFAC.AddressVelocity(MFAC.java:186)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="IssuePinExpiration" time="0.0" classname="TestNG.MFAC">
    <ignored/>
  </testcase> <!-- IssuePinExpiration -->
  <testcase name="AdditionalEnrollmentExpiration" time="0.0" classname="TestNG.MFAC">
    <ignored/>
  </testcase> <!-- AdditionalEnrollmentExpiration -->
  <testcase name="DetermineLockoutTime" time="0.0" classname="TestNG.MFAC">
    <ignored/>
  </testcase> <!-- DetermineLockoutTime -->
  <testcase name="AdditionalEnrollmentExpirationValidate" time="0.0" classname="TestNG.MFAC">
    <ignored/>
  </testcase> <!-- AdditionalEnrollmentExpirationValidate -->
  <testcase name="IssuePinExpirationValidate" time="0.0" classname="TestNG.MFAC">
    <ignored/>
  </testcase> <!-- IssuePinExpirationValidate -->
</testsuite> <!-- MFAC_Test -->
